ietf/rpc/factories.py

```diff
@@ -64,6 +68,26 @@ class Meta:
     desc = factory.Faker("sentence")
 
 
+class RfcToBeFactory(factory.django.DjangoModelFactory):
+    class Meta:
+        model = RfcToBe
+
+    draft = factory.SubFactory(WgDraftFactory)
+    rfc_number = factory.Sequence(lambda n: n + 1000)
+    submitted_format = SourceFormatName.objects.get(slug="xml-v3")
```

Comment:
You can avoid importing the classes from Name by using '_id' here (that is, `submitted_format_id="xml-v3"`)
etc.